# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2001-2025, Python Software Foundation
# This file is distributed under the same license as the Python package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
# Translators:
# Rafael Fontenelle <rffontenelle@gmail.com>, 2024
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Python 3.11\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-01-21 22:20+0000\n"
"PO-Revision-Date: 2023-05-24 02:15+0000\n"
"Last-Translator: Rafael Fontenelle <rffontenelle@gmail.com>, 2024\n"
"Language-Team: Russian (https://app.transifex.com/python-doc/teams/5390/"
"ru/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ru\n"
"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || "
"(n%100>=11 && n%100<=14)? 2 : 3);\n"

#: ../../library/errno.rst:2
msgid ":mod:`errno` --- Standard errno system symbols"
msgstr ""

#: ../../library/errno.rst:9
msgid ""
"This module makes available standard ``errno`` system symbols. The value of "
"each symbol is the corresponding integer value. The names and descriptions "
"are borrowed from :file:`linux/include/errno.h`, which should be all-"
"inclusive."
msgstr ""
"Этот модуль предоставляет стандартные системные символы ``errno``. Значение "
"каждого символа представляет собой соответствующее целочисленное значение. "
"Имена и описания заимствованы из файла :file:`linux/include/errno.h`, "
"который должен быть полным."

#: ../../library/errno.rst:17
msgid ""
"Dictionary providing a mapping from the errno value to the string name in "
"the underlying system.  For instance, ``errno.errorcode[errno.EPERM]`` maps "
"to ``'EPERM'``."
msgstr ""
"Словарь, обеспечивающий сопоставление значения errno с именем строки в "
"базовой системе. Например, ``errno.errorcode[errno.EPERM]`` соответствует "
"``'EPERM'``."

#: ../../library/errno.rst:21
msgid ""
"To translate a numeric error code to an error message, use :func:`os."
"strerror`."
msgstr ""
"Чтобы преобразовать числовой код ошибки в сообщение об ошибке, используйте :"
"func:`os.strerror`."

#: ../../library/errno.rst:23
msgid ""
"Of the following list, symbols that are not used on the current platform are "
"not defined by the module.  The specific list of defined symbols is "
"available as ``errno.errorcode.keys()``.  Symbols available can include:"
msgstr ""
"Из следующего списка символы, которые не используются на текущей платформе, "
"не определены модулем. Конкретный список определенных символов доступен как "
"``errno.errorcode.keys()``. Доступные символы могут включать в себя:"

#: ../../library/errno.rst:30
msgid ""
"Operation not permitted. This error is mapped to the exception :exc:"
"`PermissionError`."
msgstr ""
"Операция не разрешена. Эта ошибка сопоставлена ​​с исключением :exc:"
"`PermissionError`."

#: ../../library/errno.rst:36
msgid ""
"No such file or directory. This error is mapped to the exception :exc:"
"`FileNotFoundError`."
msgstr ""
"Данный файл или каталог отсутствует. Эта ошибка сопоставлена ​​с исключением :"
"exc:`FileNotFoundError`."

#: ../../library/errno.rst:42
msgid ""
"No such process. This error is mapped to the exception :exc:"
"`ProcessLookupError`."
msgstr ""
"Нет такого процесса. Эта ошибка сопоставлена ​​с исключением :exc:"
"`ProcessLookupError`."

#: ../../library/errno.rst:48
msgid ""
"Interrupted system call. This error is mapped to the exception :exc:"
"`InterruptedError`."
msgstr ""
"Прерванный системный вызов. Эта ошибка сопоставлена ​​с исключением :exc:"
"`InterruptedError`."

#: ../../library/errno.rst:54
msgid "I/O error"
msgstr "ошибка ввода/вывода"

#: ../../library/errno.rst:59
msgid "No such device or address"
msgstr "Нет такого устройства или адреса"

#: ../../library/errno.rst:64
msgid "Arg list too long"
msgstr "Слишком длинный список аргументов"

#: ../../library/errno.rst:69
msgid "Exec format error"
msgstr "Ошибка формата Exec"

#: ../../library/errno.rst:74
msgid "Bad file number"
msgstr "Плохой номер файла"

#: ../../library/errno.rst:79
msgid ""
"No child processes. This error is mapped to the exception :exc:"
"`ChildProcessError`."
msgstr ""
"Никаких дочерних процессов. Эта ошибка сопоставлена ​​с исключением :exc:"
"`ChildProcessError`."

#: ../../library/errno.rst:85
msgid ""
"Try again. This error is mapped to the exception :exc:`BlockingIOError`."
msgstr ""
"Попробуйте еще раз. Эта ошибка сопоставлена ​​с исключением :exc:"
"`BlockingIOError`."

#: ../../library/errno.rst:90
msgid "Out of memory"
msgstr "Недостаточно памяти"

#: ../../library/errno.rst:95
msgid ""
"Permission denied.  This error is mapped to the exception :exc:"
"`PermissionError`."
msgstr ""
"Доступ запрещен. Эта ошибка сопоставлена ​​с исключением :exc:"
"`PermissionError`."

#: ../../library/errno.rst:101
msgid "Bad address"
msgstr "Плохой адрес"

#: ../../library/errno.rst:106
msgid "Block device required"
msgstr "Требуется блочное устройство"

#: ../../library/errno.rst:111
msgid "Device or resource busy"
msgstr "Устройство или ресурс заняты"

#: ../../library/errno.rst:116
msgid ""
"File exists. This error is mapped to the exception :exc:`FileExistsError`."
msgstr ""
"Файл существует. Эта ошибка сопоставлена ​​с исключением :exc:"
"`FileExistsError`."

#: ../../library/errno.rst:122
msgid "Cross-device link"
msgstr "Связь между устройствами"

#: ../../library/errno.rst:127
msgid "No such device"
msgstr "Нет такого устройства"

#: ../../library/errno.rst:132
msgid ""
"Not a directory. This error is mapped to the exception :exc:"
"`NotADirectoryError`."
msgstr ""
"Не каталог. Эта ошибка сопоставлена ​​с исключением :exc:`NotADirectoryError`."

#: ../../library/errno.rst:138
msgid ""
"Is a directory. This error is mapped to the exception :exc:"
"`IsADirectoryError`."
msgstr ""
"Это каталог. Эта ошибка сопоставлена ​​с исключением :exc:`IsADirectoryError`."

#: ../../library/errno.rst:144
msgid "Invalid argument"
msgstr "Неверный аргумент"

#: ../../library/errno.rst:149
msgid "File table overflow"
msgstr "Переполнение файловой таблицы"

#: ../../library/errno.rst:154
msgid "Too many open files"
msgstr "Слишком много открытых файлов"

#: ../../library/errno.rst:159
msgid "Not a typewriter"
msgstr "Не пишущая машинка"

#: ../../library/errno.rst:164
msgid "Text file busy"
msgstr "текстовый файл занят"

#: ../../library/errno.rst:169
msgid "File too large"
msgstr "Слишком большой файл"

#: ../../library/errno.rst:174
msgid "No space left on device"
msgstr "На устройстве не осталось свободного места"

#: ../../library/errno.rst:179
msgid "Illegal seek"
msgstr "Нелегальный поиск"

#: ../../library/errno.rst:184
msgid "Read-only file system"
msgstr "Файловая система, доступная только для чтения"

#: ../../library/errno.rst:189
msgid "Too many links"
msgstr "Слишком много ссылок"

#: ../../library/errno.rst:194
msgid ""
"Broken pipe. This error is mapped to the exception :exc:`BrokenPipeError`."
msgstr ""
"Сломанная труба. Эта ошибка сопоставлена ​​с исключением :exc:"
"`BrokenPipeError`."

#: ../../library/errno.rst:200
msgid "Math argument out of domain of func"
msgstr "Математический аргумент выходит за рамки функции"

#: ../../library/errno.rst:205
msgid "Math result not representable"
msgstr "Математический результат не поддается представлению"

#: ../../library/errno.rst:210
msgid "Resource deadlock would occur"
msgstr "Произойдет взаимоблокировка ресурсов"

#: ../../library/errno.rst:215
msgid "File name too long"
msgstr "Слишком длинное имя файла"

#: ../../library/errno.rst:220
msgid "No record locks available"
msgstr "Нет доступных блокировок записей"

#: ../../library/errno.rst:225
msgid "Function not implemented"
msgstr "функция не реализована"

#: ../../library/errno.rst:230
msgid "Directory not empty"
msgstr "Каталог не пуст"

#: ../../library/errno.rst:235
msgid "Too many symbolic links encountered"
msgstr "Обнаружено слишком много символических ссылок"

#: ../../library/errno.rst:240
msgid ""
"Operation would block. This error is mapped to the exception :exc:"
"`BlockingIOError`."
msgstr ""
"Операция была бы заблокирована. Эта ошибка преобразована в исключение :exc:"
"`BlockingIOError`."

#: ../../library/errno.rst:246
msgid "No message of desired type"
msgstr "Нет сообщения нужного типа"

#: ../../library/errno.rst:251
msgid "Identifier removed"
msgstr "идентификатор удален"

#: ../../library/errno.rst:256
msgid "Channel number out of range"
msgstr "Номер канала вне диапазона"

#: ../../library/errno.rst:261
msgid "Level 2 not synchronized"
msgstr "Уровень 2 не синхронизирован"

#: ../../library/errno.rst:266
msgid "Level 3 halted"
msgstr "Уровень 3 остановлен"

#: ../../library/errno.rst:271
msgid "Level 3 reset"
msgstr "Сброс уровня 3"

#: ../../library/errno.rst:276
msgid "Link number out of range"
msgstr "Номер ссылки выходит за пределы диапазона"

#: ../../library/errno.rst:281
msgid "Protocol driver not attached"
msgstr "Драйвер протокола не подключен"

#: ../../library/errno.rst:286
msgid "No CSI structure available"
msgstr "Нет доступной структуры CSI"

#: ../../library/errno.rst:291
msgid "Level 2 halted"
msgstr "Уровень 2 остановлен"

#: ../../library/errno.rst:296
msgid "Invalid exchange"
msgstr "Недопустимый обмен"

#: ../../library/errno.rst:301
msgid "Invalid request descriptor"
msgstr "Недопустимый дескриптор запроса"

#: ../../library/errno.rst:306
msgid "Exchange full"
msgstr "Обмен полный"

#: ../../library/errno.rst:311
msgid "No anode"
msgstr "Без анода"

#: ../../library/errno.rst:316
msgid "Invalid request code"
msgstr "Неверный код запроса"

#: ../../library/errno.rst:321
msgid "Invalid slot"
msgstr "Недопустимый слот"

#: ../../library/errno.rst:326
msgid "File locking deadlock error"
msgstr "Ошибка взаимоблокировки при блокировке файла"

#: ../../library/errno.rst:331
msgid "Bad font file format"
msgstr "Неправильный формат файла шрифта"

#: ../../library/errno.rst:336
msgid "Device not a stream"
msgstr "Устройство не является потоком"

#: ../../library/errno.rst:341
msgid "No data available"
msgstr "Нет доступных данных"

#: ../../library/errno.rst:346
msgid "Timer expired"
msgstr "Таймер истек"

#: ../../library/errno.rst:351
msgid "Out of streams resources"
msgstr "Недостаточно ресурсов потоков"

#: ../../library/errno.rst:356
msgid "Machine is not on the network"
msgstr "Компьютер не подключен к сети"

#: ../../library/errno.rst:361
msgid "Package not installed"
msgstr "Пакет не установлен"

#: ../../library/errno.rst:366
msgid "Object is remote"
msgstr "Объект находится на удалении"

#: ../../library/errno.rst:371
msgid "Link has been severed"
msgstr "Связь была разорвана"

#: ../../library/errno.rst:376
msgid "Advertise error"
msgstr "Рекламная ошибка"

#: ../../library/errno.rst:381
msgid "Srmount error"
msgstr "Ошибка Srmount"

#: ../../library/errno.rst:386
msgid "Communication error on send"
msgstr "Ошибка связи при отправке"

#: ../../library/errno.rst:391
msgid "Protocol error"
msgstr "Ошибка протокола"

#: ../../library/errno.rst:396
msgid "Multihop attempted"
msgstr "Попытка выполнения нескольких переходов"

#: ../../library/errno.rst:401
msgid "RFS specific error"
msgstr "Специфическая ошибка RFS"

#: ../../library/errno.rst:406
msgid "Not a data message"
msgstr "Сообщение без данных"

#: ../../library/errno.rst:411
msgid "Value too large for defined data type"
msgstr "Слишком большое значение для определенного типа данных"

#: ../../library/errno.rst:416
msgid "Name not unique on network"
msgstr "Имя не уникально в сети"

#: ../../library/errno.rst:421
msgid "File descriptor in bad state"
msgstr "Файловый дескриптор в плохом состоянии"

#: ../../library/errno.rst:426
msgid "Remote address changed"
msgstr "Удаленный адрес изменен"

#: ../../library/errno.rst:431
msgid "Can not access a needed shared library"
msgstr "Не удается получить доступ к необходимой общей библиотеке"

#: ../../library/errno.rst:436
msgid "Accessing a corrupted shared library"
msgstr "Доступ к поврежденной общей библиотеке"

#: ../../library/errno.rst:441
msgid ".lib section in a.out corrupted"
msgstr "Секция .lib в файле a.out повреждена"

#: ../../library/errno.rst:446
msgid "Attempting to link in too many shared libraries"
msgstr "Попытка связать слишком много общих библиотек"

#: ../../library/errno.rst:451
msgid "Cannot exec a shared library directly"
msgstr "Невозможно запустить общую библиотеку напрямую"

#: ../../library/errno.rst:456
msgid "Illegal byte sequence"
msgstr "Недопустимая последовательность байтов"

#: ../../library/errno.rst:461
msgid "Interrupted system call should be restarted"
msgstr "Прерванный системный вызов должен быть перезапущен"

#: ../../library/errno.rst:466
msgid "Streams pipe error"
msgstr "Ошибка канала потоков"

#: ../../library/errno.rst:471
msgid "Too many users"
msgstr "Слишком много пользователей"

#: ../../library/errno.rst:476
msgid "Socket operation on non-socket"
msgstr "Работа с сокетом на не-сокете"

#: ../../library/errno.rst:481
msgid "Destination address required"
msgstr "Требуется адрес назначения"

#: ../../library/errno.rst:486
msgid "Message too long"
msgstr "Слишком длинное сообщение"

#: ../../library/errno.rst:491
msgid "Protocol wrong type for socket"
msgstr "Неправильный тип протокола для сокета"

#: ../../library/errno.rst:496
msgid "Protocol not available"
msgstr "Протокол недоступен"

#: ../../library/errno.rst:501
msgid "Protocol not supported"
msgstr "Протокол не поддерживается"

#: ../../library/errno.rst:506
msgid "Socket type not supported"
msgstr "Тип сокета не поддерживается"

#: ../../library/errno.rst:511
msgid "Operation not supported on transport endpoint"
msgstr "Операция не поддерживается на конечной транспортной точке"

#: ../../library/errno.rst:516
msgid "Operation not supported"
msgstr "Операция не поддерживается"

#: ../../library/errno.rst:523
msgid "Protocol family not supported"
msgstr "Семейство протоколов не поддерживается"

#: ../../library/errno.rst:528
msgid "Address family not supported by protocol"
msgstr "Семейство адресов не поддерживается протоколом"

#: ../../library/errno.rst:533
msgid "Address already in use"
msgstr "Адрес уже используется"

#: ../../library/errno.rst:538
msgid "Cannot assign requested address"
msgstr "Не удается назначить запрошенный адрес"

#: ../../library/errno.rst:543
msgid "Network is down"
msgstr "Сеть отключена"

#: ../../library/errno.rst:548
msgid "Network is unreachable"
msgstr "Сеть недоступна"

#: ../../library/errno.rst:553
msgid "Network dropped connection because of reset"
msgstr "Сеть прервала соединение из-за перезагрузки"

#: ../../library/errno.rst:558
msgid ""
"Software caused connection abort. This error is mapped to the exception :exc:"
"`ConnectionAbortedError`."
msgstr ""
"Программное обеспечение привело к прерыванию соединения. Эта ошибка "
"преобразована в исключение :exc:`ConnectionAbortedError`."

#: ../../library/errno.rst:564
msgid ""
"Connection reset by peer. This error is mapped to the exception :exc:"
"`ConnectionResetError`."
msgstr ""
"Соединение сбрасывается одноранговым узлом. Эта ошибка преобразована в "
"исключение :exc:`ConnectionResetError`."

#: ../../library/errno.rst:570
msgid "No buffer space available"
msgstr "Нет свободного места в буфере"

#: ../../library/errno.rst:575
msgid "Transport endpoint is already connected"
msgstr "Конечная транспортная точка уже подключена"

#: ../../library/errno.rst:580
msgid "Transport endpoint is not connected"
msgstr "Конечная транспортная точка не подключена"

#: ../../library/errno.rst:585
msgid ""
"Cannot send after transport endpoint shutdown. This error is mapped to the "
"exception :exc:`BrokenPipeError`."
msgstr ""
"Не удается отправить сообщение после завершения работы конечной транспортной "
"точки. Эта ошибка связана с исключением :exc:`BrokenPipeError`."

#: ../../library/errno.rst:591
msgid "Too many references: cannot splice"
msgstr "Слишком много ссылок: невозможно соединить"

#: ../../library/errno.rst:596
msgid ""
"Connection timed out. This error is mapped to the exception :exc:"
"`TimeoutError`."
msgstr ""
"Время ожидания соединения истекло. Эта ошибка отображается как исключение :"
"exc:`TimeoutError`."

#: ../../library/errno.rst:602
msgid ""
"Connection refused. This error is mapped to the exception :exc:"
"`ConnectionRefusedError`."
msgstr ""
"В соединении отказано. Эта ошибка сопоставлена с исключением :exc:"
"`ConnectionRefusedError`."

#: ../../library/errno.rst:608
msgid "Host is down"
msgstr "Хост отключен"

#: ../../library/errno.rst:613
msgid "No route to host"
msgstr "Нет маршрута к хосту"

#: ../../library/errno.rst:618
msgid ""
"Operation already in progress. This error is mapped to the exception :exc:"
"`BlockingIOError`."
msgstr ""
"Операция уже выполняется. Эта ошибка преобразована в исключение :exc:"
"`BlockingIOError`."

#: ../../library/errno.rst:624
msgid ""
"Operation now in progress. This error is mapped to the exception :exc:"
"`BlockingIOError`."
msgstr ""
"В настоящее время выполняется операция. Эта ошибка сопоставлена с "
"исключением :exc:`BlockingIOError`."

#: ../../library/errno.rst:630
msgid "Stale NFS file handle"
msgstr "Устаревший дескриптор файла NFS"

#: ../../library/errno.rst:635
msgid "Structure needs cleaning"
msgstr "Структура нуждается в очистке"

#: ../../library/errno.rst:640
msgid "Not a XENIX named type file"
msgstr "Не файл именованного типа XENIX"

#: ../../library/errno.rst:645
msgid "No XENIX semaphores available"
msgstr "Семафоры XENIX недоступны"

#: ../../library/errno.rst:650
msgid "Is a named type file"
msgstr "Это файл именованного типа"

#: ../../library/errno.rst:655
msgid "Remote I/O error"
msgstr "Ошибка удаленного ввода-вывода"

#: ../../library/errno.rst:660
msgid "Quota exceeded"
msgstr "Превышена квота"

#: ../../library/errno.rst:664
msgid "Interface output queue is full"
msgstr "Очередь вывода интерфейса заполнена"

#: ../../library/errno.rst:670
msgid ""
"Capabilities insufficient. This error is mapped to the exception :exc:"
"`PermissionError`."
msgstr ""
"Возможности недостаточны. Эта ошибка сопоставлена ​​с исключением :exc:"
"`PermissionError`."

#: ../../library/errno.rst:673
msgid ":ref:`Availability <availability>`: WASI, FreeBSD"
msgstr ":ref:`Доступность <доступность>`: WASI, FreeBSD"

#: ../../library/errno.rst:680
msgid "Operation canceled"
msgstr "Операция отменена"

#: ../../library/errno.rst:687
msgid "Owner died"
msgstr "Владелец умер"

#: ../../library/errno.rst:694
msgid "State not recoverable"
msgstr "Состояние не подлежит восстановлению"
